app/src/main/res/values/strings.xml

```diff
@@ -141,7 +139,7 @@
     <string name="iv">IV</string>
     <string name="percent">%d%%</string>
     <string name="unknown_percent">"?%"</string>
-    <string name="percent_phrase">Percent phrase</string>
-    <string name="iv_range_phrase">IV range phrase</string>
-    <string name="higher_values">Higher values</string>
+    <string name="percent_phrase">Overall Appraisal:</string>
+    <string name="iv_range_phrase">Best Stat(s) Appraisal:</string>
+    <string name="higher_values">Best Stat(s):</string>
```

Comment:
I think percent_phrase, iv_range_phrase and higher_values are unused currently... gotta check.